apps/web/src/__tests__/translations.test.ts

```diff
@@ -96,6 +96,18 @@ const whitelist = [
   'Buy Squad / Bunnies',
   'Swap Token',
   "Your browser doesn't support iframe",
+  'Weekly revenue sharing',
+  'Weekly CAKE pool rewards',
+  'Boost rewards on your favorite farms',
+  'Claim additional incentives from other protocols',
+  'Boost your farm earnings',
+  'Supports multi-chain boosts',
+  'Use your veCAKE to vote on important governance proposals',
+  'Participate in IFO public sales',
+  'Supports multi-chain IFOs',
+  'Boost your winning in Trading Rewards campaigns. Regardless which chain you are trading on.',
+  'Boost your earnings in fixed staking.',
+  'More to come...',
```

Comment:
Why add in here?

Reply:
Because it dynamically invokes the translation function, the unit test identifies it as an unused string.

![image](https://github.com/pancakeswap/pancake-frontend/assets/109973128/44779a49-b1af-416c-8aa7-4da3b73f5523)
![image](https://github.com/pancakeswap/pancake-frontend/assets/109973128/cf98ccbe-99bb-4424-8514-8458aec8240a)

Reply:
hmm we need to figure out a better way to extract dynamic translations

Reply:
Change it to [<Trans>123</Trans>].
U can check the code in `IfoQuestions` config.

Exp:
`description: [<Trans>The CAKE from the participation fee will be burnt as part of the weekly token burn.</Trans>],`

Reply:
If we use switch to use `ReactNode` for desc, just be aware that we need to use a different value for the `key` prop of li. Key only accepts `string | number`

Reply:
Fixed